### PR TITLE
Document stale PR policy

### PR DESCRIPTION
**Pull request by aaltay**
Related dev list discussion:

https://lists.apache.org/thread.html/f1b899b33c7e314e77bc904d7b4ab64e15f6ce8eb0b87c3efab9de81@%3Cdev.beam.apache.org%3E

R: @lukecwik

### CODE REVIEW

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/beam_PreCommit_Website_Stage/665/

Jenkins built the site at commit id 950c0c498debce3549a2b00999da140c953c16fa with Jekyll and staged it [here](http://apache-beam-website-pull-requests.storage.googleapis.com/309/index.html). Happy reviewing.

Note that any previous site has been deleted. This staged site will be automatically deleted after its TTL expires. Push any commit to the pull request branch or re-trigger the build to get it staged again.

**Comment by aaltay** [comment]
retest this please

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/beam_PreCommit_Website_Stage/666/

Jenkins built the site at commit id 950c0c498debce3549a2b00999da140c953c16fa with Jekyll and staged it [here](http://apache-beam-website-pull-requests.storage.googleapis.com/309/index.html). Happy reviewing.

Note that any previous site has been deleted. This staged site will be automatically deleted after its TTL expires. Push any commit to the pull request branch or re-trigger the build to get it staged again.

File: src/contribute/contribution-guide.md [hunk 1]
```diff
@@ -301,6 +301,10 @@ From another local branch, run:
 	$ git branch -d <my-branch>
 	$ git push <GitHub_user> --delete <my-branch>
 
+### Stale pull requests
+
+The community will close stale pull requests in order to keep the project healthy. A pull request becomes stale after its author fails to respond to actionable comments for 60 days.  Author of a closed pull request is welcome to reopen the same pull request again in the future. The associated JIRAs will be unassigned from the authors but will stay open.
```
Comment (lukecwik) [inline src/contribute/contribution-guide.md]: `will be unassigned from the authors` -> `will be unassigned from the author`

Comment (aaltay) [reply src/contribute/contribution-guide.md]: Done.

**Comment by aaltay** [comment]
dead link is unrelated - being fixed in https://github.com/apache/beam-site/pull/314

**Comment by aaltay** [comment]
retest this please

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/beam_PreCommit_Website_Stage/675/

Jenkins built the site at commit id 605fbf632ae247f0f724927f2ca6e94548ce79bd with Jekyll and staged it [here](http://apache-beam-website-pull-requests.storage.googleapis.com/309/index.html). Happy reviewing.

Note that any previous site has been deleted. This staged site will be automatically deleted after its TTL expires. Push any commit to the pull request branch or re-trigger the build to get it staged again.

**Comment by lukecwik** [comment]
@asfgit merge

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/beam_PreCommit_Website_Stage/676/

Jenkins built the site at commit id 605fbf632ae247f0f724927f2ca6e94548ce79bd with Jekyll and staged it [here](http://apache-beam-website-pull-requests.storage.googleapis.com/309/index.html). Happy reviewing.

Note that any previous site has been deleted. This staged site will be automatically deleted after its TTL expires. Push any commit to the pull request branch or re-trigger the build to get it staged again.

**Comment by asfgit** [comment]
MergeBot: Merge: PR 309 in state error with description Unexpected Error: 502 Server Error: Proxy Error for url: https://builds.apache.org/job/beam_PreCommit_Website_Merge/api/python?tree=builds%5Bnumber%2Curl%5D.. URL: http://mergebot-vm.apache.org/beam-website/309.

**Comment by asfgit** [comment]
Error: MergeBot encountered an unexpected error while processing this PR: 502 Server Error: Proxy Error for url: https://builds.apache.org/job/beam_PreCommit_Website_Merge/api/python?tree=builds%5Bnumber%2Curl%5D.

**Pull request closed without merging**